src/common/search-store.ts

```diff
@@ -36,28 +36,28 @@ export class SearchStore {
   }
 
   /**
-   * Does searching within text array, create a list of search keyword occurences.
-   * Each keyword "occurency" is saved as index of the the line where keyword founded
+   * Does searching within text array, create a list of search keyword occurrences.
+   * Each keyword "occurrence" is saved as index of the the line where keyword founded
```

Comment:
While you're fixing typos...
```suggestion
   * Each keyword "occurrence" is saved as index of the line where keyword found
```